app/src/main/res/values-vi/strings.xml

```diff
@@ -0,0 +1,205 @@
+    <string name="price">Giá</string>
+    <string name="update_success">CẬp nhật thành công</string>
+    <string name="top_shop">Cửa hàng nổi bật</string>
+    <string name="categories">Thể loại</string>
+    <string name="top_product">Sản phẩm nổi bật</string>
+    <string name="create_product_successful">Tạo sản phẩm thành công!</string>
+    <string name="edit_product">Chỉnh sửa sản phẩm</string>
+    <string name="update_successful">Cập nhật thành công</string>
+    <string name="note">Note :</string>
+    <string name="order_successful">Đặt hàng thành công!</string>
+    <string name="accepted_status">accepted</string>
+    <string name="rejected_status">rejected</string>
+    <string name="input_your_note">Nhập ghi chú</string>
+    <string name="add_note">Thêm ghi chú</string>
+    <string name="reject_all">Từ chối tất cả</string>
+    <string name="accept_all">Chấp nhận tất cả</string>
+    <string name="leave">Rời khỏi</string>
+    <string name="close_order">Đóng đơn hàng</string>
+    <string name="close_order_success">Đóng đơn hàng thành công</string>
+    <string name="done_orders">Kết thúc đơn hàng</string>
+    <string name="rejected_orders">Từ chối đơn hàng</string>
+    <string name="name_domain">Tên miền</string>
+    <string name="secret">secret</string>
+    <string name="professed">professed</string>
+    <string name="add_domain">Thêm miền</string>
+    <string name="add_domain_success">Thêm miền thành công!</string>
+    <string name="quantity_order_history">Số lượng: </string>
+    <string name="msg_delete_domain">Bạn có thật sự muốn xóa miền này?</string>
+    <string name="msg_leave_domain">Bạn thực sự muốn rời miền?</string>
+    <string name="delete_the_domain_success">Xóa miền thành công!</string>
+    <string name="leave_the_domain_successful">Rời khỏi miền thành công!</string>
+    <string name="edit_domain">Chỉnh sửa miền</string>
+    <string name="add_to_cart_success">Thêm vào giỏ hàng thành công!</string>
+    <string name="add_to_cart">Thêm vào giỏ hàng</string>
+    <string name="order_now">Đặt ngay</string>
+    <string name="hint_notes">Bất kỳ ý kiến như ít dừa, ít ngọt có thể được đề xuất ở đây...</string>
+    <string name="quick_order">Đặt nhanh</string>
+    <string name="list_domain">Danh sách miền</string>
+    <string name="list_order">Danh sách đơn hàng</string>
+    <string name="manager_in_shop">Quản lý</string>
+    <string name="close">Đóng</string>
+    <string name="see_manager">Xem quản lý</string>
+    <string name="user_in_domain">Thành viên trong miền</string>
+    <string name="delete">Xóa</string>
+    <string name="set_manager">Thiết lập quản lý</string>
+    <string name="delete_success">Xóa thành công!</string>
+    <string name="msg_delete_user">Bạn có thực sự muốn xóa thành viên này?</string>
+    <string name="change_role_success">Thay đổi quyền vụ thành công!</string>
+    <string name="view_user">Xem thành viên</string>
+    <string name="view_product">Xem Sản phẩm</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.view_product` appears to be unused
 Unused resources make applications larger and slow down builds.